cmake/CMakeLists.txt

```diff
@@ -400,15 +400,14 @@ if (onnxruntime_USE_MKLDNN OR onnxruntime_USE_MKLML)
 endif()
 
 if (onnxruntime_USE_MKLML)
-  add_definitions(-DUSE_MKLML=1)
-  # USE_MKML_FOR_BLAS may cause numerical differences in tests so disable by default
-  #add_definitions(-DUSE_MKLML_FOR_BLAS=1)
+  add_definitions(-DUSE_MKLML=1 -DUSE_MKLML_FOR_BLAS=1)
   if (WIN32 OR APPLE)
     list(APPEND onnxruntime_EXTERNAL_LIBRARIES mklml)
   else()
     list(APPEND onnxruntime_EXTERNAL_LIBRARIES mklml_intel)
   endif()
   list(APPEND onnxruntime_EXTERNAL_DEPENDENCIES project_mklml)
+  include_directories(${MKLML_INCLUDE_DIR})
```

Comment:
Why do you move the ${MKLML_INCLUDE_DIR} from the onnxruntime_providers.cmake to here?

Reply:
Not only mkldnn needs it. All the projects that uses math.h needs it.